app/static/js/event_list.js

```diff
@@ -41,3 +41,16 @@ function removeRsvpForEvent(eventID){
 
   })
 }
+
+document.getElementById("toggleButton").addEventListener("click", function() {
+  var tableRows = document.getElementsByClassName("showlist");
```

Comment:
We should be doing this the jquery way like all of our other code, not the regular javascript way